src/train.py

```diff
@@ -175,7 +176,7 @@ def train():
     print ('Model statistics saved to {}.'.format(
       os.path.join(FLAGS.train_dir, 'model_metrics.txt')))
 
-    saver = tf.train.Saver(tf.global_variables())
+    saver = tf.train.Saver(tf.global_variables(), max_to_keep=MAX_MODEL_TO_KEEP)
```

Comment:
I do not think this is very useful in practice. Better sticking with the default of max_to_keep=5 and add keep_checkpoint_every_n_hours=1 so we have a checkpoint every 1 hour in addition to the rolling window of 5 recent checkpoints.